cdap-ui/app/cdap/components/NamespaceDropdown/index.js

```diff
@@ -240,7 +241,9 @@ export default class NamespaceDropdown extends Component {
               <small>{T.translate('features.Navbar.NamespaceDropdown.namespaceLabel')}</small>
               <span>{currentNamespace}</span>
             </div>
-            <span className="fa fa-caret-down float-xs-right" />
+            <span className="float-xs-right">
+              <SVG icon="icon-caret-down" />
```

Comment:
If we can wrap things inside a span with a class name do we need a `className` prop for `SVG`? We should probably follow one pattern